outputwriter.java

```diff
@@ -77,6 +90,10 @@ public static String createEssay() {
 				linee = buffer.readLine();
 			}
 			thirdTransitionWords = liness.toArray(new String[liness.size()]);
+
+      Random rand = new Random();
```

Comment:
If you initialize two Random objects (see line 97), they either have to be given different names, or you have to use the same object. Preferably the latter. So all you have to do (if I'm thinking correctly) is erase line 97 and leave your line 94 in.